src/test/java/com/wikia/webdriver/common/core/XMLReader.java

```diff
@@ -21,13 +20,17 @@ private XMLReader() {
    */
   public static String getValue(File file, String key) {
 
+    if (!file.exists() || file.isDirectory()) {
+      RuntimeException e = new ConfigurationRuntimeException("Cannot find a file with credentials");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately throw this expression instead of assigning it to the temporary variable "e". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1488)